src/consts.h

```diff
@@ -19,7 +19,7 @@
 #define   VERSION            52000          // Eventually will be deprecated. 
 #define   SEMVERSION_MAJOR   "5"            // Major Semantic Version
 #define   SEMVERSION_MINOR   "2"            // Minor Semantic Version
-#define   SEMVERSION_PATCH   "0.dev5"      // Patch Semantic Version
+#define   SEMVERSION_PATCH   "0.dev6"      // Patch Semantic Version
```

Comment:
I feel like I may have this wrong - should I keep this at `0.dev5`?

Reply:
@aerispaha, i'd say keep it at dev5. We will make this next release with @jennwuu's code.